pkg/connection/store/vault/store.go

```diff
@@ -169,7 +170,7 @@ func (ss *SecretStore) DeleteKeyValues(_ context.Context, s *store.Secret, do ..
 	}
 
 	for _, o := range do {
-		if err = o(context.Background(), s); err != nil {
+		if err = o(ctx, s); err != nil {
```

Comment:
@turkenh I presume it's safe to pass this context on here. Was not passing it intentional?

Reply:
I don’t remember that it was intentional, so looks good.